Package_SON.pretty/VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm.kicad_mod

```diff
@@ -0,0 +1,44 @@
+(module VSON-10-1EP_3x3mm_P0.5mm_EP2.1x1mm (layer F.Cu) (tedit 5B9F93A0)
+  (descr http://rohmfs.rohm.com/en/techdata_basic/ic/package/vson010v3030_1-e.pdf)
```

Comment:
I would recommend replacing the packaging datasheet (currently listed) with the full datasheet (found [here](http://rohmfs.rohm.com/en/products/databook/datasheet/ic/power/switching_regulator/bd8314nuv-e.pdf)). The packaging datasheet doesn't have any information about the actual package size, only the land pattern.